### PR TITLE
docs: link directly to metioned functions

### PR DESCRIPTION
**Pull request by kirillmorozov**
Updates links in mentoring guidance for ISBN Verifier exercise on Python track to point directly to mentioned built-in functions.

Expand tab in python code block into 4 spaces.

### CODE REVIEW

**Review by SleeplessByte — CHANGES_REQUESTED** [review]
Can you use the same link syntax for the new links?

```markdown
[builtin-function-sum]: https://docs.python.org/3/library/functions.html#sum
```

etc.

**Comment by kirillmorozov** [comment]
@SleeplessByte, I've made changes that you've requested but @safwansamsudeen was quicker and already merged #2260.

My PR goes one step further as it have links directly to specific functions. Should I rebase my fork and continue or #2260 is enough?

**Comment by SleeplessByte** [comment]
I like your changes! If you like your changes too, i recommend rebasing and reapplying.

I'll happily merge it right after.

**Review by SleeplessByte — DISMISSED** [review]

File: tracks/python/exercises/isbn-verifier/mentoring.md [hunk 1]
```diff
@@ -101,12 +101,13 @@ for position, value in enumerate(isbn):
         try:
             total += int(value) * (10 - position)
         except ValueError:
-	    return False
```
Comment (IsaacG) [inline tracks/python/exercises/isbn-verifier/mentoring.md:104]: Is this meant to be in this PR? It looks like there's no code in the `except` path.

Comment (kirillmorozov) [reply tracks/python/exercises/isbn-verifier/mentoring.md:104]: Looks like one more indentation level is required.

Comment (kirillmorozov) [reply tracks/python/exercises/isbn-verifier/mentoring.md:104]: There was a tab symbol and it seems visually it was okay, I've swapped it to 4 spaces but it wasn't enough.

**Review by IsaacG — APPROVED** [review]

**Comment by SleeplessByte** [comment]
> Damn, made a typo in PR title...

Don't worry about it 🙃

**Pull request merged**